certif/app/styles/pages/authenticated/sessions/import.scss

```
@@ -96,10 +59,26 @@
     }
   }
 
+  &__section--download__instruction {
+    color: $pix-neutral-50;
+    font-size: 14px;
+
+    ul {
+      list-style: disc;
+      margin-top: 6px;
+      margin-left: 24px;
+    }
+
+    > p {
```

Avis perso : Pas fan des sélecteur globaux sur des balises HTML qui peut générer des effets non désirés (je ne connais pas les pratiques sur le projet). En tous cas avec la classe .pix-message--warning ça marche pas mal et on cible notre message d'avertissement scoper dans notre container :)

```suggestion
   .pix-message--warning {
```